src/services/note/delete.ts

```diff
@@ -29,6 +29,10 @@ export default async function(user: User, note: Note, quiet = false) {
 		Notes.decrement({ id: note.renoteId }, 'score', 1);
 	}
 
+	if (note.replyId) {
+		await Notes.decrement({ id: note.replyId }, 'repliesCount', 1);
```

Comment:
`await` する必要は無さそうです。
```suggestion
		Notes.decrement({ id: note.replyId }, 'repliesCount', 1);
```

Reply:
いやこれ逆にdecrement全部awaitしないといけないんじゃ

Reply:
`await` is required, otherwise the Promise may not be executed because it is deleted by the garbage collector.

https://stackoverflow.com/questions/61834275/what-happens-to-the-promise-if-we-dont-await-for-it

Reply:
> `await` する必要は無さそうです。

意図した部分を汲み取ると、もしかして decrement 全部を配列に push していって最後に Promise.all した方が良い？